bugbug/models/regressor.py

```diff
@@ -127,9 +129,14 @@ def __init__(
                 ("union", ColumnTransformer(column_transformers)),
             ]
         )
-
-        self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
-        self.clf.set_params(predictor="cpu_predictor")
+        base_clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+        base_clf.set_params(predictor="cpu_predictor")
+        if calibration:
+            self.clf = IsotonicRegressionCalibrator(base_clf)
+            # This is a temporary workaround for the error : "Model type not yet supported by TreeExplainer"
+            self.calculate_importance = False
+        else:
+            self.clf = base_clf
```

Comment:
Nit: You could simplify this code by doing:
```suggestion
        self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
        self.clf.set_params(predictor="cpu_predictor")

        if calibration:
            self.clf = IsotonicRegressionCalibrator(self.clf)
            # This is a temporary workaround for the error : "Model type not yet supported by TreeExplainer"
            self.calculate_importance = False
```